src/Apps/Sell/Components/SubmissionStepTitle.tsx

```diff
@@ -0,0 +1,9 @@
+import { Text } from "@artsy/palette"
+
+export const SubmissionStepTitle: React.FC<{ title: string }> = ({ title }) => {
+  return (
+    <Text variant={["lg-display", "xl"]} mb={2}>
+      {title}
+    </Text>
+  )
+}
```

Comment:
Both versions are okay, but I would prefer this one, which is a bit more standard React.


```suggestion
export const SubmissionStepTitle: React.FC = ({ children }) => {
  return (
    <Text variant={["lg-display", "xl"]} mb={2}>
      {children}
    </Text>
  )
}
```

Reply:
Agree, looks better 👍